api/proto/disperser/disperser.proto

```diff
@@ -15,6 +15,7 @@ service Disperser {
 	// RESOURCE_EXHAUSTED (429): request is rate limited for the quorum specified in the error msg.
 	//                           user should retry after the specified duration.
 	// INTERNAL (500): serious error, user should NOT retry.
+	// UNAVAILABLE (503): intermittent failure, user can retry after 1s & exponential backoff.
```

Comment:
I remember in the proxy, 503 corresponds to "fallback to ETH", meaning it is less of a transient error. 500 corresponds to some other error. It is bit counter-intuitive

Reply:
Ya I agree......... it's hard here because what we're trying to do with the fallback is not really typical Http behavior, so doesn't fit very well into any grpc/http codes. Here I was following the grpc codes https://grpc.io/docs/guides/status-codes/
<img width="958" alt="image" src="https://github.com/user-attachments/assets/c74a49aa-506d-4cc9-875c-b390259f52d0">
which mention that 500 is serious error. Guess it doesn't mean that we shouldn't retry though.... But also 503 here means "you can retry with exponential backoff", which is kind of annoying because it goes against what we're trying to do with the 503 failover in batcher->proxy communication. Perhaps we need to be more precise and failover on 503 that has a `RETRY-AFTER` header of >= 1 hour? We're getting into complex, annoying, and bug-prone territory for implementers though...